### PR TITLE
Ability to measure angles

### PR DESCRIPTION
**Pull request by supersonicclay**

### CODE REVIEW

**Comment by xintongxia** [comment]
what is the use case for this mode?

File: modules/edit-modes/src/lib/measure-angle-mode.js [hunk 1]
```diff
@@ -0,0 +1,130 @@
+// @flow
+
+import turfBearing from '@turf/bearing';
+import turfCenter from '@turf/center';
+import memoize from 'memoizee';
```
Comment (xintongxia) [inline modules/edit-modes/src/lib/measure-angle-mode.js]: There is a small util file in deck which supports memorize, and avoid introduce another dependency.

https://github.com/uber/deck.gl/blob/master/modules/core/src/utils/memoize.js

File: modules/edit-modes/src/lib/measure-angle-mode.js [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+export class MeasureAngleMode extends BaseGeoJsonEditMode {
+  _getTooltips = memoize((modeConfig, vertex, point1, point2) => {
+    let tooltips = DEFAULT_TOOLTIPS;
+
+    if (vertex && point1 && point2) {
+      const { formatTooltip, measurementCallback } = modeConfig || {};
+      const units = 'deg';
+
+      // const angle = turfAngle(point1, vertex, point2);
```
Comment (xintongxia) [inline modules/edit-modes/src/lib/measure-angle-mode.js]: delete line 27?

**Comment by supersonicclay** [comment]
> what is the use case for this mode?

We had a request for several measurement modes #289.

**Pull request merged**